src/hapi/pipelines/database/org.py

```diff
@@ -1,21 +1,41 @@
 import logging
+from typing import Dict
 
 from hapi_schema.db_org import DBOrg
+from hdx.location.names import clean_name
+from hdx.scraper.utilities.reader import Read
 from sqlalchemy import select
 from sqlalchemy.orm import Session
 
-from hapi.pipelines.database.base_uploader import BaseUploader
+from .base_uploader import BaseUploader
 
 logger = logging.getLogger(__name__)
 
 
 class Org(BaseUploader):
-    def __init__(self, session: Session):
+    def __init__(
+        self,
+        session: Session,
+        datasetinfo: Dict[str, str],
+    ):
         super().__init__(session)
+        self._datasetinfo = datasetinfo
         self.data = {}
+        self._org_map = {}
 
     def populate(self):
-        raise NotImplementedError()
+        logger.info("Populating org mapping")
+        reader = Read.get_reader()
+        headers, iterator = reader.get_tabular_rows(
+            self._datasetinfo["url"],
+            headers=2,
+            dict_form=True,
+            format="csv",
+            file_prefix="org",
```

Comment:
@mcarans I added the file prefixes because it seemed that they would be saved with the prefix of the scraper that was last run, so they were getting a "food_security" prefix.

Reply:
Thanks for reporting this. I have fixed this bug in HDX Python Scraper 2.2.5. You may be able to remove the prefix (default is ""), although it would also be fine to keep it if it helps distinguish the download.

Reply:
These files get the names of a google sheet download so I will leave the prefixes so we know what they are.